### DISCUSSION TITLE
Pythonコードブロック内文字列に ... が含まれているとそれ以降の文法解釈・色付けがおかしくなる

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#519] [要望・アイデア・不具合]
下図のようにPythonコードブロック内の文字列に `...` が含まれていると、それ以降の文法解釈・色付けがおかしくなります。

![image](https://github.com/increments/qiita-discussions/assets/4151943/7ae9b4af-f595-42d3-86bd-6c5f9bc2282e)

**Comment by getty104** [comment]
@shiracamus
ご不便おかけしてしまい申し訳ございません、こちら修正したのでご確認お願いします！
![スクリーンショット 2023-08-07 11 44 45](https://github.com/increments/qiita-discussions/assets/20335778/4c8a58d5-1ee5-400d-b39e-630d1da072a9)

**Reply by shiracamus** [reply]
修正されたことを確認しました。
対応ありがとうございました。
